### ISSUE TITLE
Error in coxph(Surv(OS.time, OS) ~ RS, x) :    data contains an infinite predictor

### ISSUE THREAD

**Issue by linjianghuaye** [l-magnificence/Mime#62]
您好，最近我在使用Mime1这个包进行机器学习分析的时候，突然出现一个报错：Error in coxph(Surv(OS.time, OS) ~ RS, x) : 
  data contains an infinite predictor，麻烦各位大神看看这是为什么。

**Comment by 771086341** [comment]
我也遇到的相同的问题

**Comment by Aivence** [comment]
同，请问是否解决

**Comment by l-magnificence** [comment]
see #44

**Comment by 771086341** [comment]
我的问题得到了解决，我修改了seed，注意不要设置过大

**Comment by CC-DrDU** [comment]
For anyone come across this issue and has checked their own data making sure that no infinite or NA value was found, my solution is that the OS in the training set and the validation set **MUST BE** numeric (I came across this issue even though the OS was integer). We can simply use the as.numeric function to do that.

**Comment by CC-DrDU** [comment]
I came across this issue so many times even though I'm pretty sure that there is no NA or infinite value in my dataset. However, after filtering my dataset according to the correlations between variables (1720 -> 380), I solved the problem. I did not use the log normalization but use z-scale instead, so I don't think this issue is about the normalization method. Maybe the number of variables matters...
